tests/playground.html

```diff
@@ -1060,6 +1060,71 @@ <h1>Blockly Playground</h1>
       <block type="empty_block"></block>
       <block type="empty_block_with_mutator"></block>
     </category>
+    <category name="Drag">
+      <label text="Drag each of to the workspace"></label>
```

Comment:
"Drag each of to the workspace" looks like it's missing a word or has too many words.  Please fix.